Ensembl/ensembl-variation
# Fix `Config.pm` to properly auto-generate `Constants.pm`

`Constants.pm` has been manually edited for a few releases, but it should be auto-generated based on `Config.pm` by running:
```bash
perl create_config_consts.pl --config Bio::EnsEMBL::Variation::Utils::Config > Constants.pm
```

## Changes
* Update `Config.pm` and `create_config_consts.pl` to reflect manual changes made to `Constants.pm`
* Update `Config.pm` to separate `@EXPORT_OK` and `%EXPORT_TAGS` with newlines (easier to check for differences in future changes)
* Re-generate `Constants.pm` to include missing exported fields:
  * `SO_TERM_SPLICE_DONOR_5TH_BASE_VARIANT`
  * `SO_TERM_SPLICE_DONOR_REGION_VARIANT`
  * `SO_TERM_SPLICE_POLYPYRIMIDINE_TRACT_VARIANT`

## Testing

1. Run `create_config_consts.pl` to auto-generate `Constants.pm` (should result in an identical `Constants.pm` file to the one in this PR)
2. Run VEP with some sample data to ensure that `Constants.pm` is properly formatted
